### PR TITLE
Add naive implementation of cluster

### PR DESCRIPTION
**Pull request by ChuckLangford**
This commit allows for a naive implementation of Node's clustering. It
also allows the app to be run without clustering for ease of debugging
(this is the default).

### CODE REVIEW

**Comment by ChuckLangford** [comment]
@johnrhampton Want thoughts on how readable this is. I made clustering optional because when I have previously used it, it would have been nice for it to be easily disabled for debugging.

**Comment by johnrhampton** [comment]
@ChuckLangford seems very readable and clean.  Really like how easy it is to disable clustering.

File: config/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 const config = {
   env: process.env.NODE_ENV,
   type: process.env.PROCESS_TYPE,
+  enableCluster: process.env.ENABLE_CLUSTER ? process.env.ENABLE_CLUSTER.toLowerCase() === 'true' : false,
```
Comment (johnrhampton) [inline config/index.js]: we should probably create a helper function for this behavior

Comment (ChuckLangford) [inline config/index.js]: Hold your thoughts until you see today's PR. These are going to conflict a bit.

**Pull request merged**